htdocs/core/lib/images.lib.php

```diff
@@ -321,7 +321,8 @@ function dolRotateImage($file_path)
 function correctExifImageOrientation($fileSource, $fileDest, $quality = 95)
 {
 	if (function_exists('exif_read_data') ) {
-		$exif = exif_read_data($fileSource);
+		$exif = @exif_read_data($fileSource); // Of course, suppressing warnings is not a good idea, but there doesn't seem to be a way of avoiding this warning: there's no function to test the validity of the EXIF data without trying to read it and getting the warning.
```

Comment:
@atm-john is it possible to catch exception error?

Reply:
yes but it mean test on different php version to check if "try catch" work perfectly (based on a bad experience)
but I cant do that before next week so I do a quick fix...